op-acceptance-tests/tests/interop/proofs/proposer_test.go

```diff
@@ -0,0 +1,68 @@
+package proofs
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lmittmann/w3"
+)
+
+var (
+	gameCountFn        = w3.MustNewFunc("gameCount()", "uint256")
+	gameAtIndexFn      = w3.MustNewFunc("gameAtIndex(uint256)", "uint32, uint64, address")
+	rootClaimFn        = w3.MustNewFunc("rootClaim()", "bytes32")
+	l2SequenceNumberFn = w3.MustNewFunc("l2SequenceNumber()", "uint256")
+)
+
+func TestProposer(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewSimpleInterop(t)
+
+	deployment := sys.L2Networks()[0].Escape().Deployment()
+	disputeGameFactoryAddr := deployment.DisputeGameFactoryProxyAddr()
+
+	l1Client := sys.L1EL.Escape().EthClient()
```

Comment:
(nit/optional) Apparently these `Escape()` methods are considered a crutch and we're supposed to be adding utilities we need to the  `op-devstack` interfaces.